### PR TITLE
feat(cli): adding sync repo and sync repos to CLI

### PR DESCRIPTION
**Pull request by ecrupper**
Creating the capability to run the new API command to sync repositories between Vela and SCM in the CLI. Currently using the format: `vela sync repo --o <org> --r <repo>` and `vela sync repo --o <org> --all` for sync and syncAll respectively. Open to other ideas on that front. I figured having only one letter difference (`sync repo` and `sync repos`) is a tad worrying. 

Related PRs: [SDK-Go](https://github.com/go-vela/sdk-go/pull/137) and [Server](https://github.com/go-vela/server/pull/539)

### CODE REVIEW

**Review by KellyMerrick — DISMISSED** [review]
lgtm

File: command/repo/sync.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		return err
+	}
+
+	// check if the all flag has been set
+	if c.Bool("all") {
+		// create the repo configuration
+		//
+		// https://pkg.go.dev/github.com/go-vela/cli/action/repo?tab=doc#Config
+		r := &repo.Config{
+			Action: "syncAll",
```
Comment (jbrockopp) [inline command/repo/sync.go:87]: Should this be a constant that is added to `github.com/go-vela/cli/internal`?

https://github.com/go-vela/cli/blob/85ac81f95e15d40701c6ea4496a3cf5b1a3e4444/internal/internal.go#L128-L174

File: command/repo/sync.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		//
+		// https://pkg.go.dev/github.com/go-vela/cli/action/repo?tab=doc#Config.SyncAll
+		return r.SyncAll(client)
+	}
+
+	// create the repo configuration
+	//
+	// https://pkg.go.dev/github.com/go-vela/cli/action/repo?tab=doc#Config
+	r := &repo.Config{
+		Action: "sync",
```
Comment (jbrockopp) [inline command/repo/sync.go:107]: Should this be a constant that is added to `github.com/go-vela/cli/internal`?

https://github.com/go-vela/cli/blob/85ac81f95e15d40701c6ea4496a3cf5b1a3e4444/internal/internal.go#L128-L174

**Review by jbrockopp — CHANGES_REQUESTED** [review]

**Review by KellyMerrick — DISMISSED** [review]
lgtm

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by KellyMerrick — APPROVED** [review]
lgtm

**Pull request merged**